govc/USAGE.md

```diff
@@ -36,338 +36,339 @@ but appear via `govc $cmd -h`:
- - [sso.idp.ldap.update](#ssoidpldapupdate)
- - [sso.idp.ls](#ssoidpls)
- - [sso.lpp.info](#ssolppinfo)
- - [sso.lpp.update](#ssolppupdate)
- - [sso.service.ls](#ssoservicels)
- - [sso.user.create](#ssousercreate)
- - [sso.user.id](#ssouserid)
- - [sso.user.ls](#ssouserls)
- - [sso.user.rm](#ssouserrm)
- - [sso.user.update](#ssouserupdate)
- - [storage.policy.create](#storagepolicycreate)
- - [storage.policy.info](#storagepolicyinfo)
- - [storage.policy.ls](#storagepolicyls)
- - [storage.policy.rm](#storagepolicyrm)
- - [tags.attach](#tagsattach)
- - [tags.attached.ls](#tagsattachedls)
- - [tags.category.create](#tagscategorycreate)
- - [tags.category.info](#tagscategoryinfo)
- - [tags.category.ls](#tagscategoryls)
- - [tags.category.rm](#tagscategoryrm)
- - [tags.category.update](#tagscategoryupdate)
- - [tags.create](#tagscreate)
- - [tags.detach](#tagsdetach)
- - [tags.info](#tagsinfo)
- - [tags.ls](#tagsls)
- - [tags.rm](#tagsrm)
- - [tags.update](#tagsupdate)
- - [task.cancel](#taskcancel)
- - [tasks](#tasks)
- - [tree](#tree)
- - [vapp.destroy](#vappdestroy)
- - [vapp.power](#vapppower)
- - [vcsa.access.consolecli.get](#vcsaaccessconsolecliget)
- - [vcsa.access.consolecli.set](#vcsaaccessconsolecliset)
- - [vcsa.access.dcui.get](#vcsaaccessdcuiget)
- - [vcsa.access.dcui.set](#vcsaaccessdcuiset)
- - [vcsa.access.shell.get](#vcsaaccessshellget)
- - [vcsa.access.shell.set](#vcsaaccessshellset)
- - [vcsa.access.ssh.get](#vcsaaccesssshget)
- - [vcsa.access.ssh.set](#vcsaaccesssshset)
- - [vcsa.log.forwarding.info](#vcsalogforwardinginfo)
- - [vcsa.net.proxy.info](#vcsanetproxyinfo)
- - [vcsa.shutdown.cancel](#vcsashutdowncancel)
- - [vcsa.shutdown.get](#vcsashutdownget)
- - [vcsa.shutdown.poweroff](#vcsashutdownpoweroff)
- - [vcsa.shutdown.reboot](#vcsashutdownreboot)
- - [version](#version)
- - [vm.change](#vmchange)
- - [vm.clone](#vmclone)
- - [vm.console](#vmconsole)
- - [vm.create](#vmcreate)
- - [vm.customize](#vmcustomize)
- - [vm.destroy](#vmdestroy)
- - [vm.disk.attach](#vmdiskattach)
- - [vm.disk.change](#vmdiskchange)
- - [vm.disk.create](#vmdiskcreate)
- - [vm.guest.tools](#vmguesttools)
- - [vm.info](#vminfo)
- - [vm.instantclone](#vminstantclone)
- - [vm.ip](#vmip)
- - [vm.keystrokes](#vmkeystrokes)
- - [vm.markastemplate](#vmmarkastemplate)
- - [vm.markasvm](#vmmarkasvm)
- - [vm.migrate](#vmmigrate)
- - [vm.network.add](#vmnetworkadd)
- - [vm.network.change](#vmnetworkchange)
- - [vm.option.info](#vmoptioninfo)
- - [vm.power](#vmpower)
- - [vm.question](#vmquestion)
- - [vm.rdm.attach](#vmrdmattach)
- - [vm.rdm.ls](#vmrdmls)
- - [vm.register](#vmregister)
- - [vm.unregister](#vmunregister)
- - [vm.upgrade](#vmupgrade)
- - [vm.vnc](#vmvnc)
- - [volume.ls](#volumels)
- - [volume.rm](#volumerm)
- - [volume.snapshot.create](#volumesnapshotcreate)
- - [volume.snapshot.ls](#volumesnapshotls)
- - [volume.snapshot.rm](#volumesnapshotrm)
- - [vsan.change](#vsanchange)
- - [vsan.info](#vsaninfo)
+- [govc usage](#govc-usage)
```

Comment:
Oh - did we miss a regen from a prior change? Maybe 48bdfada6aa. Looks like the only change here is the first `[govc usage](#govc-usage)` line, and then that caused an extra space ident to be added.

Reply:
That was auto format of vscode adding a section from above (treating this toc as toc for the whole file but it is just for the below parts).